_includes/sqlstyle.guide.md

```diff
@@ -111,13 +88,13 @@ SELECT first_name
 
 ```sql
 SELECT first_name AS fn
-  FROM staff AS s1
+FROM staff AS s1
```

Comment:
i think all of us are kind of against the idea of the "river" - i made a comment somewhere about how it ends up privileging `AND` and `OR` the same as `WHERE`, and everyone else just thinks that they will realistically not write their queries that way.